adafruit_wsgi/wsgi_app.py

```diff
@@ -66,8 +66,17 @@ def __call__(self, environ: Dict[str, str], start_response: Callable):
 
         if match:
             args, route = match
-            status, headers, resp_data = route["func"](request, *args)
-
+            response_params = route["func"](request, *args)
+            if (
+                not isinstance(response_params, (list, tuple))
+                or len(response_params) != 3
+            ):
+                raise RuntimeError(
+                    "Proper HTTP response return not given for request handler '{}'".format(
+                        route["func"].__name__
+                    )
+                )
+            status, headers, resp_data = response_params
```

Comment:
Another more compact way to do this, using the errors that would be thrown:
```py
>> a,b,c = (1,2,3,4)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: too many values to unpack (expected 3)
>>> a,b,c = 7
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'int' object is not iterable
>>> 
```

```suggestion
            try:
                status, headers, resp_data = route["func"](request, *args)
            except ValueError, TypeError:
                raise RuntimeError(
                    "Proper HTTP response return not given for request handler '{}'".format(
                        route["func"].__name__
```

Reply:
I always forget you can catch multiple exceptions.  I think this is perfect!